static/src/components/PlayerDisplay.js

```diff
@@ -62,113 +57,205 @@ function mapDispatchToProps(dispatch) {
 
 export class PlayerDisplay extends Component {
 
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
+        console.log("constructor");
         this.state = {
-            polling: false,
+            displayID: undefined,
```

Comment:
Best practice is to initialise as null not undefined. This allows us to easily check in the render method the variable for example:
```
{ this.state.displayID && 
<h1>Hello World</h1>
}
```
Where hello world is only displayed if this.state.displayID is not null